### PR TITLE
[MM-21122] Update plugins.json

### PR DESCRIPTION
**Pull request by iomodo**
#### Summary
This PR will add to `plugins.json` file:
Zoom version 1.2.0 
Custom Attributes version 1.1.0
Jira version 2.3.2
Autolink version 1.1.2
Antivirus version 0.1.2

And Remove:
aws-SNS version 0.1.0
#### Ticket Link

  Fixes https://mattermost.atlassian.net/browse/MM-21122
  Fixes https://mattermost.atlassian.net/browse/MM-21335
  Fixes https://mattermost.atlassian.net/browse/MM-21665
  Fixes https://mattermost.atlassian.net/browse/MM-21698
  Fixes https://mattermost.atlassian.net/browse/MM-21699
  Fixes: https://mattermost.atlassian.net/browse/MM-21692

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -369,6 +414,128 @@
     },
     "updated_at": "2019-06-11T14:04:45Z"
   },
+  {
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-zoom",
+    "icon_data": "",
```
Comment (hanzei) [inline plugins.json]: Not related to this PR, but given how popular the zoom plugin is, it should have an icon.

cc @asaadmahmood

File: plugins.json [hunk 2]
```diff
@@ -210,6 +210,51 @@
     },
     "updated_at": "2019-10-03T17:39:18Z"
   },
+  {
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-autolink",
+    "icon_data": "",
```
Comment (hanzei) [inline plugins.json]: Same for the autolink plugin

**Comment by lieut-data** [comment]
This PR also fixes https://mattermost.atlassian.net/browse/MM-21692, since it updates `plugin.json` after having marked the older versions of the plugins with changed plugin ids as pre-release.

**Comment by hanzei** [comment]
@lieut-data Are you sure? The "bad" JIRA release 1.0.2 is still part of `plugin.json` (https://github.com/mattermost/mattermost-marketplace/blob/MM-21122-Update-plugins-json/plugins.json#L2388-L2448)

**Comment by lieut-data** [comment]
Thanks, @hanzei -- not sure where I misjudged here, but I've since marked that release as pre-release and have pushed a change to this PR to eliminate same. It also updates a plugin signature now available for GitHub 0.11.0.

**Pull request merged**